### DISCUSSION TITLE
Sanity Check On A Few Things

### DISCUSSION THREAD

**Discussion by DrAnonymous** [forkineye/ESPixelStick#430] [General]
After a hiatus of getting a basic light setup working last year, I've gotten the hardware back out and am trying to make sure I understand the latest state of things. I'd appreciate some clarification/corrections on the following:

- The V3 firmware for the ESP8266 is the latest release from last year (Nov. I believe)
- The V4 beta FW works on the ESP8266 and the ESP32
- There's no ESPixelStick for the ESP32 right now, so a person needs to make their own circuit to drive an LED string (Is there a block diagram or schematic showing what needs to be done?)
- A light show can now be stored on a flashdisk in FSEQ format with a V4 firmware. This would let a person make a standalone setup, versus having to stream a show using the E1.31 network interface.

Am I understanding the current state of things properly? Feedback is much appreciated!

**Comment by DrAnonymous** [comment]
After trying the new FW, I believe the answer to my original questions was yes.

**Comment by forkineye** [comment]
You had it correct. For the FSEQ files, they must be V2 uncompressed and sparse if in a show with other controllers. FPP Connect in xLights will generate and upload the FSEQ files for you. If in Vixen 3, you need to export a FSEQ per controller.  The default output pin mappings for ESP32 are in the input/output section of the wiki.

**Reply by DrAnonymous** [reply]
Thanks for following up. Sorry for the delay in getting back to this, as I was on work travel.
